### PR TITLE
Adds a basic CSS Reset and Open Sans font

### LINKED ISSUE #104: Add additional styling setup - CSS Reset / Font

### PR DESCRIPTION
**Pull request by limbovski**
closes #104

### CODE REVIEW

File: src/variables.ts [hunk 1]
```diff
@@ -10,6 +10,6 @@ export enum Color {
 
 export const fontWeights = {
   regular: 400,
-  medium: 500,
-  bold: 600,
+  medium: 600,
+  bold: 700,
```
Comment (MoSattler) [inline src/variables.ts:14]: why the change here?

Comment (SimonNord) [reply src/variables.ts:14]: Yeah that´s the only input I have too. According to [this](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) font weight normal should be 500

Comment (SimonNord) [reply src/variables.ts:14]: oh wait, I see now.
The imported weights are 400/600/700. Can you import 400, 500, 600 instead?

Comment (limbovski) [reply src/variables.ts:14]: Yo. So in standard CSS implementation bold is 700 and there is no Open Sans available with a weight of 500. So on stead of 400, 500, 600 we need to go with 400, 600, 700.

Comment (SimonNord) [reply src/variables.ts:14]: well, not much of a different right? It´s okey for me to use the Open Sans 400, 600, 700
